python/CMakeLists.txt

```
@@ -54,6 +55,7 @@ target_link_libraries(_pycudaq PUBLIC nvqir
 
 install(TARGETS _pycudaq DESTINATION ${CMAKE_INSTALL_PREFIX})
 install(FILES cudaq.py DESTINATION ${CMAKE_INSTALL_PREFIX})
+install(FILES apps/chemistry.py DESTINATION ${CMAKE_INSTALL_PREFIX}/apps/)
```

This causes the build to fail due to an incorrect path during installation (CUDA Quantum can be built and tested successfully, but won't be installed correctly, hence the unit tests passing but the packaging validation failing):

```suggestion
install(FILES domains/chemistry.py DESTINATION ${CMAKE_INSTALL_PREFIX}/apps/)
```